tests/VolleyManagement.UnitTests/Services/TeamService/TeamBuilder.cs

```diff
@@ -78,9 +82,25 @@ public TeamBuilder WithAchievements(string achievements)
         /// </summary>
         /// <param name="captainId">Test team captain</param>
         /// <returns>Team builder object</returns>
-        public TeamBuilder WithCaptain(int captainId)
+        public TeamBuilder WithCaptain(PlayerId captainId)
         {
-            _team.Captain = captainId;
+            _team.SetCaptain(captainId);
+            return this;
+        }
+
+        /// <summary>
+        /// Sets test team roster
+        /// </summary>
+        /// <param name="roster">Test team roster</param>
+        /// <returns>Team builder object</returns>
+        public TeamBuilder WithRoster(IEnumerable<PlayerId> roster)
+        {
+            _team = new Team(_team.Id,
```

Comment:
Why do you create new team here?

Reply:
Because in this method we define roster itself, we do not add it to current. Or is it best to get rid of it?